src/components/RenderRouter/ListItem.tsx

```diff
@@ -1132,6 +1135,12 @@ class ListItem extends React.Component<Props, State> {
       return null;
     }
   }
+
+  renderInstaTile(item: any): JSX.Element | null { // Instagram tile styling
+    if(!item) return null;
+    return <div className="ListInstagramTile"><img width={this.state.windowWidth < 375 ? 120 :this.state.windowWidth < 600 ? 156 : 264} height={this.state.windowWidth < 375 ? 120 :this.state.windowWidth < 600 ? 156 : 264} src={item.thumbnails[2].src}></img></div>
```

Comment:
Why not do this in CSS? Also, use variable width so the images scale with screen size.